tiledb/sm/query/query.h

```diff
@@ -42,6 +42,7 @@
 #include "tiledb/sm/query/writer.h"
 #include "tiledb/sm/storage_manager/storage_manager.h"
 
+#include <tiledb/sm/misc/logger.h>
```

Comment:
Can this be added to the above list in double-quotes? Then `make format` will alphabetize it.